dashboard/src/modules/components/OverviewComponent/NewRunsComponent.jsx

```diff
@@ -148,7 +148,7 @@ const NewRunsComponent = () => {
               const isItemFavorited = !!item?.metadata?.[USER_FAVORITE];
               const isItemSeen = !!item?.metadata?.[DASHBOARD_SEEN];
               return (
-                <Tbody key={rowIndex}>
+                <Tbody key={rowIndex} isExpanded={isRunExpanded(item)}>
```

Comment:
So, new runs are "expandable", but saved runs and expiring runs are not?

Reply:
> Can you help me understand how you arrived at that conclusion?

I compared the line here to the [analogous line](https://github.com/distributed-system-analysis/pbench/blob/18465a897ce573cd648b9788b0191da2bc9b791a/dashboard/src/modules/components/OverviewComponent/SavedRunsComponent.jsx#L115) in the saved-run code:  the `isExpanded` prop is being added here but not there, so I figured I would inquire about the apparent lack of symmetry.